wit/world.wit

```diff
@@ -3,4 +3,5 @@ package wasi:clocks@0.2.0;
 world imports {
     import monotonic-clock;
     import wall-clock;
+    import timezone;
```

Comment:
ah shoot, I forgot to add the `@unstable` gate in my PR. We should change this to be:
```suggestion
    @unstable(feature = clocks-timezone)
    import timezone;
```